### PR TITLE
iOS: Updated doc with 24.7.1 changes

### PR DESCRIPTION
**Pull request by ijunaid**

### CODE REVIEW

File: ios/current.md [hunk 1]
```diff
@@ -728,7 +728,8 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
   <strong>Enterprise Edition Feature</strong>
   <p>
     This feature is only available with an
-    <a href="https://count.ly/enterprise-edition">Enterprise Edition</a> subscription.
+    <a href="https://count.ly/enterprise-edition">Enterprise Edition</a> and built-in
```
Comment (arifBurakDemiray) [inline ios/current.md:731]: please also change enterprie link

File: ios/current.md [hunk 2]
```diff
@@ -2434,7 +2467,8 @@ Countly.sharedInstance().askForNotificationPermission(options: authorizationOpti
   <strong>Enterprise Edition Feature</strong>
   <p>
     This feature is only available with an
-    <a href="https://count.ly/enterprise-edition">Enterprise Edition</a> subscription.
```
Comment (arifBurakDemiray) [inline ios/current.md:2437]: countly.com/enterprise

File: ios/current.md [hunk 3]
```diff
@@ -3308,7 +3342,8 @@ aFeedbackWidget.recordResult(nil) // if user dismissed the feedback widget witho
   <strong>Enterprise Edition Feature</strong>
   <p>
     This feature is only available with an
-    <a href="https://count.ly/enterprise-edition">Enterprise Edition</a> subscription.
+    <a href="https://count.ly/enterprise-edition">Enterprise Edition</a> and built-in
```
Comment (arifBurakDemiray) [inline ios/current.md:3345]: same here

**Review by arifBurakDemiray — CHANGES_REQUESTED** [review]
cuple de changes

File: ios/current.md (hunk 2 above)
Comment (ijunaid) [reply ios/current.md:2437]: nice catch 👍

**Review by arifBurakDemiray — APPROVED** [review]

File: ios/current.md [hunk 4]
```diff
@@ -1619,8 +1620,41 @@ Countly.sharedInstance().changeDeviceIDWithoutMerge("new_device_id")</code></pre
...
+  </div>
+  <div class="tab">
+    <pre><code class="objectivec">[Countly.sharedInstance enableTemporaryDeviceIDMode];<br></code></pre>
+  </div>
+  <div class="tab is-hidden">
+    <pre><code class="swift">Countly.sharedInstance().enableTemporaryDeviceIDMode()</code></pre>
+  </div>
+</div>
+<div class="callout callout--warning">
+  <p>For SDK version 24.7.0 you should use the below methods:</p>
```
Comment (turtledreams) [inline ios/current.md:1654]: lets remove the old methods from this docs and lets add a link to an older doc for those. we can mention the methods here are available from 24.7.1 and prior to that version they should check out the link.

**Review by turtledreams — CHANGES_REQUESTED** [review]

**Review by turtledreams — APPROVED** [review]

**Pull request merged**